Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM ubuntu:latest
+RUN apt update
```

Comment:
If you have to run apt update, you also want to clean the apt cache after installing the packages to reduce the size of the resulting image.